wolfcrypt/src/random.c

```diff
@@ -1479,6 +1480,16 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
 int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 {
+#ifdef WOLF_CRYPTO_CB
+    int ret;
+
+    if (os != NULL && os->devId != INVALID_DEVID) {
+        ret = wc_CryptoCb_RandomSeed(os, output, sz);
+        if (ret != NOT_COMPILED_IN)
```

Comment:
If the error is NOT_COMPILED_IN is silently ignoring correct.  That is, someone turned on WOLF_CRYPTO_CB expected to use the callback, we detect a compile time error at runtime and silently switch to software.  Seems like we could do better?

Reply:
I would expect to see NOT_COMPILED_IN returned when other CryptoCb algorithms are supported but not RandomSeed.
The PKCS#11 implementation only returns NOT_COMPILED_IN if WC_NO_RNG is defined, which means it will never be called in the first place unless something wacky is happening.
Am I missing something? This is the same pattern used in other places.

Reply:
Ok, thanks.